mappings/net/minecraft/SharedConstants.mapping

```diff
@@ -30,6 +30,7 @@ CLASS net/minecraft/class_155 net/minecraft/SharedConstants
 	METHOD method_34872 setGameVersion (Lnet/minecraft/class_6489;)V
 		ARG 0 gameVersion
 	METHOD method_36208 createGameVersion ()V
+	METHOD method_43250 optimizeDataFixer ()V
```

Comment:
This doesn't perform any data fixer optimization:

```java
public static void optimizeDataFixer() {
    dataFixerPhase = switch (dataFixerPhase) {
        case DataFixerPhase.INITIALIZED_UNOPTIMIZED -> throw new IllegalStateException("Tried to enable datafixer optimization after unoptimized initialization");
        case DataFixerPhase.INITIALIZED_OPTIMIZED -> DataFixerPhase.INITIALIZED_OPTIMIZED;
        default -> DataFixerPhase.UNINITIALIZED_OPTIMIZED;
    };
}
```

Could it be named `enableDataFixerOptimization` or `updateDataFixerPhase` instead?

Reply:
```suggestion
	METHOD method_43250 enableDataFixerOptimization ()V
```